Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -94,6 +134,23 @@ namespace AzToolsFramework
             QRect itemGeometry(rect);
             itemGeometry.setRight(itemCount == 2 ? itemGeometry.width() - perItemWidth : perItemWidth);
             itemGeometry.setLeft(itemGeometry.left() + (m_depth * indentSize));
+
+            if (m_showExpander)
+            {
+                if (!m_expanderWidget)
+                {
+                    m_expanderWidget = new QCheckBox(parentWidget());
+                    m_expanderWidget->setCheckState(m_expanded ? Qt::Checked : Qt::Unchecked);
+                    AzQtComponents::CheckBox::applyExpanderStyle(m_expanderWidget);
+                    connect(m_expanderWidget, &QCheckBox::stateChanged, this, &DPELayout::onCheckstateChanged);
```

Comment:
sorry, I don't understand. Which part should be a helper?

Reply:
The whole 4 lines. I think it would make it more readable since those 4 lines are effectively just creating the expander (so it could be a helper like `CreateExpanderWidget()`) and it would be safer to group that operation in case it needs to be reused.

Reply:
I usually only do that if/when I reuse a code block, but sure, I can do that.